atlite/cutout.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, path, **cutoutparams):
             Time range to include in the cutout, e.g. "2011" or
             ("2011-01-05", "2011-01-25")
             This is necessary when building a new cutout.
-        bounds : GeoSeries.bounds | DataFrame, optional
+        bounds : GeoSeries.bounds | DataFrame | Tuple, optional
             The outer bounds of the cutout or as a DataFrame
             containing (min.long, min.lat, max.long, max.lat).
```

Comment:
```suggestion
        bounds : DataFrame | Tuple, optional
            The outer bounds of the cutout containing (min.long, min.lat, max.long, max.lat)
            or a single-row DataFrame with [["minx", "miny", "maxx", "maxy"]] column values.
            From GeoPandas DataFrames and Series, this is easily accessible through `.geometry.bounds`.
```

I checked their documentation again and noticed that `bounds` are not a class, only a tuple attribute. I've moved it into the docstring.